raiden-dapp/src/services/raiden-service.ts

```diff
@@ -371,5 +328,3 @@ export class RaidenInitializationFailed extends Error {}
 export class FindRoutesFailed extends Error {}
 
 export class PFSRequestFailed extends Error {}
-
-export class MintTokenFailed extends Error {}
```

Comment:
Nice catch!

Reply:
Btw I know I started doing this `Error` thing in the first place but in a lot of cases it doesn't make that much sense. And where it makes we could also use a separate try/catch in order to have the same result. We should certainly re-evaluate and reduce that.